### PR TITLE
[Flickr] Update

### PR DESCRIPTION
**Pull request by thongphp**
app/Console/Commands/Flickr/FlickrContacts.php
- [x] Please rename Flickr facade to FlickrClient

app/Jobs/Flickr/FlickrContact.php

- [x] Validate NSID is valid before process handle
- [x] If user is deleted or banned by Flickr ( we can't get detail ). Please make sure job still process as finished instead throw to retry.
- [x] If we need retry by any reason make sure we do clear cache for this specific request. Or next time will load from cached.
Please also let me know what's purpose of status ? I would use state instead

app/Console/Commands/Flickr/FlickrPhotos.php

- [x] Only process 1 contact. Or you'll flood server with tons of queues
- [x] FlickrValidator to validate NSID
- [x] For 2 sub queues also validate $nsid before any process.

app/Jobs/Flickr/FlickrSyncToGooglePhoto.php

- [x] Split download & upload to 2 jobs

Other

- [x] Make sure do not double query by use touch() and save() after
- [x] Rename GooglePhotoFacade to GooglePhotoClient

### CODE REVIEW

File: config/app.php [hunk 1]
```diff
@@ -255,7 +255,7 @@
         'Es' => Elasticquent\ElasticquentElasticsearchFacade::class,
         'GoogleDrive' => GoogleDriveFacade::class,
         'GooglePhoto' => GooglePhotoFacade::class,
-        'Flickr' => Flickr::class,
+        'Flickr' => FlickrClient::class,
```
Comment (jooservices) [inline config/app.php:258]: Not sure but i assume it should be 'FlickrClient' ?

File: config/app.php [hunk 2]
```diff
@@ -255,7 +255,7 @@
...
         'flickr\urlextractor' => Flickr\UrlExtractor::class,
```
Comment (jooservices) [inline config/app.php:259]: I think it should be FlickrUrlExtractor ?

Comment (thongphp) [reply config/app.php]: I already removed this, this is alias, it not necessary since we always use App\Facades\...

File: config/app.php (hunk 1 above)
Comment (thongphp) [reply config/app.php:258]: This is alias for class, I intent to remove it in next commit, since we use App\Facade\FlickrClient

Comment (thongphp) [reply config/app.php]: I would change to FlickrClient

File: config/app.php (hunk 2 above)
Comment (thongphp) [reply config/app.php]: I would change to FlickrUrlExtractor

File: app/Console/Commands/Flickr/FlickrContact.php [hunk 3]
```diff
@@ -36,9 +36,11 @@ final class FlickrContact extends BaseCommand
      */
     public function fully(): bool
     {
-        $contact = app(ContactRepository::class)->getItemByCondition([
+        if (!$contact = app(ContactRepository::class)->getItemByCondition([
```
Comment (jooservices) [inline app/Console/Commands/Flickr/FlickrContact.php:39]: Should we use "getItemByConditions"

File: app/Console/Commands/Flickr/FlickrContacts.php [hunk 4]
```diff
@@ -56,9 +53,7 @@ public function fully(): bool
         }
 
         for ($page = 2; $page <= $contacts->contacts->pages; $page++) {
-            if (!$nextContacts = Flickr::getContactsOfCurrentUser($page)) {
-                continue;
-            }
+            $nextContacts = FlickrClient::getContactsOfCurrentUser($page);
```
Comment (jooservices) [inline app/Console/Commands/Flickr/FlickrContacts.php:56]: Not sure why we do we remove ! condition here ?

**Pull request merged**